system/extensions/esp32/wifi.toit

```diff
@@ -99,14 +113,19 @@ class WifiServiceDefinition extends NetworkServiceDefinitionBase:
       throw "wifi channel must be between 1 and 13"
     broadcast/bool := config.get wifi.CONFIG_BROADCAST --if_absent=: true
 
-    module ::= (state_.up: WifiModule.ap this ssid password broadcast channel) as WifiModule
-    if not module.ap:
+    if module_ != null:
```

Comment:
This doesn't seem right. The NetworkState object is supposed to keep the network up as long as there are users.

Reply:
Yes, the module_ may be closed here, I modify to check `state_.module`.